_data/nav.yml

```diff
@@ -1,10 +1,34 @@
 - title: "Home"
   href: "/"
 
-- title: "About"
+
+- title: Attend
+  subcategories:
+    - subtitle: Schedule
+      subhref: /schedule
+    - subtitle: Tickets
+      subhref: /tickets
+
+- title: "Sponsors & Friends"
+  subcategories:
+    - subtitle: "Sponsors"
+      subhref: "/sponsors"
+    - subtitle: "Friends"
+      subhref: "/friends"
+
+- title: "Volunteers & Presenters"
```

Comment:
I don't think we should be advertising the CFP on the website at all. Presenters can go to /cfp if they want, but it shouldn't be in the nav.

Reply:
Removed.